lib/wallet/http.js

```diff
@@ -767,9 +768,40 @@ class HTTP extends Server {
     this.get('/wallet/:id/tx/history', async (req, res) => {
       const valid = Validator.fromRequest(req);
       const acct = valid.str('account');
-      const txs = await req.wallet.getHistory(acct);
+      const reverse = valid.bool('reverse', false);
+      const limit = valid.u32('limit', 10);
+      const after = valid.brhash('after');
+      const date = valid.str('date');
+      let time = valid.u32('time');
+
+      if (date && !time) {
+        time = util.time(date);
+        enforce(time, 'Invalid date.');
+      }
+
+      enforce(limit <= this.options.maxTxs,
```

Comment:
Good idea on using this as DOS prevention. Can you add a comment explaining that because I am sure people will run into this `enforce` line without realizing that they need to set a config option `maxTxs`. It may also be desirable to turn this off for customers that require this. Its unlikely that they will exceed `Number.isSafeInteger`, but its not the easiest option to know how to configure when you effectively want it off. Just set it to a really large number and hope it doesn't eventually break?

Reply:
This is for the limit of a specific result and in practice it really should not every need to be modified or changed, however it is a configurable none-the-less. You would never want to turn the maximum off, as this would defeat the purpose and lead to CPU and memory exhaustion issues. More results can be queried in the next result.

Reply:
This can be added to configuration documentation, to describe the option.